code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -283,7 +283,7 @@
 
 /obj/item/weapon/nullrod/whip
 	name = "holy whip"
-	desc = "What a terrible night to be on Space Station 13."
+	desc = "What a terrible night to be on the Cyberiad!"
```

Comment:
@Fox-McCloud I've been away from home for some time.

I don't want to just web-editor this one as I'd like to test it first